omero/developers/Web.rst

```diff
@@ -15,6 +15,8 @@ to provide modular web tools, such as the webclient, webgateway and
 the JSON api app. This modular framework
 makes it possible to extend OMERO.web with your own apps.
 
+Since vesion ``5.14.0``, ``OMERO.web`` requires an update of Django to ``3.2.x``.  Plugin developers should read the `Guide <https://github.com/ome/omero-web/blob/master/MIGRATION_TO_DJANGO_32_GUIDE.md>`_ detailing how to migrate their plugin(s) to Django 3.2.x.
```

Comment:
```suggestion
Since version ``5.14.0``, ``OMERO.web`` requires an update of Django to ``3.2.x``.  Plugin developers should read the `Guide <https://github.com/ome/omero-web/blob/master/MIGRATION_TO_DJANGO_32_GUIDE.md>`_ detailing how to migrate their plugin(s) to Django 3.2.x.
```

Also since Django will be a transitive dependency for developers, maybe `required an update of Django` a should be rephased as `depends on Django 3.2` ?